static/js/browser2/browser_page.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import { ArcsSection } from "./arcs_sections";
+
+interface BrowserPagePropType {
+  dcid: string;
+  nodeName: string;
+}
+
+export class BrowserPage extends React.Component<BrowserPagePropType> {
+  render(): JSX.Element {
+    return (
+      <div>
```

Comment:
nit: the surrounding div aren't necessary if there's only one element. or if you just need a container, <><elem1 /><elem2 /></> works too

Reply:
done